shared/src/domain-logic/opportunityFields.test.ts

```diff
@@ -0,0 +1,240 @@
+import { generateMunicipality, generateProfileData } from "../test";
+import { getFieldsForProfile, isEveryFundingFieldAnswered } from "./opportunityFields";
+
+describe("opportunityFields", () => {
+  describe("getFieldsForProfile", () => {
+    it(
```

Comment:
[pebble] this test and the ones below that are related to `business type of nexus and a legal structure that is a trade name` can be grouped into a describe block to make it easier to read.  

``` 
describe("when business type is nexus and legal structure is a trade name ", () => {
it("returns only date of formation and home based business keys when location is not in NJ"){...}
....
})
```

Reply:
yea i grouped them all together